### PR TITLE
Microservices demo

### PR DESCRIPTION
**Pull request by AntonioDiTuri**

### CODE REVIEW

**Comment by dipankardas011** [comment]
This PR contains the usage of https://fluxcd.io/flux/cheatsheets/oci-artifacts/#helm-oci
for helm-oci (for microservice-demo by google) deploy using flux

File: clusters/microservices-demo.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  namespace: default
+spec:
+  interval: 10m
+  chart:
+    spec:
+      chart: onlineboutique
+      sourceRef:
+        kind: HelmRepository
+        name: onlineboutique
+  releaseName: onlineboutique
```
[the remark this answers is not in the document]
Comment (AntonioDiTuri) [reply clusters/microservices-demo.yaml:25]: Yeah we can drop it for the moment

**Comment by AntonioDiTuri** [comment]
I committed all the requested changes, can we close this? @rossf7 I requested your review :)

File: clusters/microservices-demo.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (nikimanoledaki) [inline clusters/microservices-demo.yaml:1]: Changed the repo structure to the following:
```
tree clusters
clusters
├── projects
│   └── falco
│       └── falco-namespace.yaml
├── base
│   ├── kepler-grafana.yaml
│   ├── kepler.yaml
│   └── kube-prometheus-stack.yaml
└── flux-system
    ├── gotk-components.yaml
    ├── gotk-sync.yaml
    └── kustomization.yaml
```
 Could move the config to `clusters/projects/falco` please? Thank you!!

File: clusters/microservices-demo.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  namespace: flux-system
+spec:
+  interval: 24h
+  type: oci
+  url: oci://us-docker.pkg.dev/online-boutique-ci/charts
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: onlineboutique
```
Comment (nikimanoledaki) [inline clusters/microservices-demo.yaml:15]:
```suggestion
  name: online-boutique
```
Nit: Could you update all the names to use kebab-case please?

**Review by nikimanoledaki — APPROVED** [review]
Some requested changes then LGTM 🥳

**Comment by AntonioDiTuri** [comment]
@rossf7 I adjusted the changes, please let me know if something else need to be reviewed

File: clusters/projects/falco/microservices-demo.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+spec:
+  interval: 24h
+  type: oci
+  url: oci://us-docker.pkg.dev/online-boutique-ci/charts
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: online-boutique
+  namespace: flux-system
```
Comment (nikimanoledaki) [inline clusters/projects/falco/microservices-demo.yaml:16]:
```suggestion
  namespace: falco
```

**Review by rossf7 — APPROVED** [review]
> I adjusted the changes, please let me know if something else need to be reviewed

@AntonioDiTuri This looks good just needs the namespace change suggested by Niki here https://github.com/cncf-tags/green-reviews-tooling/pull/13/files#r1471463552

File: clusters/projects/falco/microservices-demo.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: onlineboutique
```
Comment (nikimanoledaki) [inline clusters/projects/falco/microservices-demo.yaml:5]:
```suggestion
  name: online-boutique
```

File: clusters/projects/falco/microservices-demo.yaml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  namespace: falco
+spec:
+  interval: 24h
+  chart:
+    spec:
+      chart: online-boutique
+      sourceRef:
+        kind: HelmRepository
+        name: onlineboutique
+  releaseName: onlineboutique
```
Comment (nikimanoledaki) [inline clusters/projects/falco/microservices-demo.yaml:25]:
```suggestion
        name: online-boutique
  releaseName: online-boutique
```

File: docs/microservices_demo.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
+# How to install it manually
+
+1. Get access to the dev-cluster (ask Tag/WG leads)
+
+2. run the following command: 
+
+```
+helm upgrade onlineboutique oci://us-docker.pkg.dev/online-boutique-ci/charts/onlineboutique --install
```
Comment (nikimanoledaki) [inline docs/microservices_demo.md:8]:
```suggestion
helm upgrade online-boutique oci://us-docker.pkg.dev/online-boutique-ci/charts/online-boutique --install
```

File: docs/microservices_demo.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+```
+3. to uninstall 
+
+```
+helm delete onlineboutique
```
Comment (nikimanoledaki) [inline docs/microservices_demo.md:13]:
```suggestion
helm delete online-boutique
```

File: clusters/projects/falco/microservices-demo.yaml [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+  targetNamespace: default
```
Comment (nikimanoledaki) [inline clusters/projects/falco/microservices-demo.yaml:26]:
```suggestion
```

**Review by nikimanoledaki — APPROVED** [review]

**Pull request merged**